<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultyMapIconsPage</class>
 <widget class="QWidget" name="DifficultyMapIconsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="NoMapIcons">
     <property name="text">
      <string>No map icons</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoPlayerIcon">
     <property name="text">
      <string>No player icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoFogOfWarIcons">
     <property name="text">
      <string>No fog of war icons</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoMinimapPath">
     <property name="text">
      <string>No minimap path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoIcons">
     <property name="text">
      <string>No icons</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
